src/modules/reducers/exchange.js

```diff
@@ -11,7 +11,6 @@ export const EXCHANGE_REFRESH_ERROR = "exchange/EXCHANGE_REFRESH_ERROR";
 export const EXCHANGE_REFRESH_SUCCESS = "exchange/EXCHANGE_REFRESH_SUCCESS";
 
 const initialState = {
-    contract: null,
```

Comment:
why was this removed?

Reply:
It's just an accidental.
I set it back.